src/build_matrix_A.py

```diff
@@ -68,20 +65,20 @@ def build_matrix_A(material_property_library, mesh, time):
         #This line will create a weighted value of rho c for the boundary node
         rho_c_weighted = (volume_pre_node*rho[mat-1]*c[mat-1]+volume_post_node*rho[mat]*c[mat])/volume
         
-        A[i+1,i+2] = -k[mat]*Dt*4*np.pi/(rho_c_weighted)/volume*(1/(1/mesh[mat][1][0]-1/mesh[mat][1][1]))
-        A[i+1,i+1] = (k[mat-1]*Dt*4*np.pi/(rho_c_weighted)/volume*(1/(1/mesh[mat-1][1][prev_mesh_length-2]
+        A[i+1,i+2] = -k[mat]*4*np.pi/(rho_c_weighted)/volume*(1/(1/mesh[mat][1][0]-1/mesh[mat][1][1]))
+        A[i+1,i+1] = (k[mat-1]*4*np.pi/(rho_c_weighted)/volume*(1/(1/mesh[mat-1][1][prev_mesh_length-2]
```

Comment:
Is this now `A[i+1,i+1] = -A[i+1,i] - A[i+1, i+2]`??

Reply:
Cleaning up our code is a great idea but this is a low priority for changes currently. Thanks for the feedback.